var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -32,6 +32,7 @@ class SuperluDist(Package):
     homepage = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/"
     url = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_4.1.tar.gz"
 
+    version('xsdk-0.1.0', git='https://github.com/xiaoyeli/superlu_dist', commit='0b5369f')
```

Comment:
maybe call it `develop` for time being, otherwise `xsdk-0.1.0` will always compare less-than with any other released version. In other words, it will not satisfy conditions like `depends_on(superlu-dist@5.0.0:)`.